### PR TITLE
new package: compton 0.1.2

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by a-schaefers** [comment]
By the way the associated PR's for this, being:
libev
libconfig
libxdg-basedir
uthash

Are all good to go.

File: community/compton/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export NO_DBUS=1
+
+# fake asciidoc
+touch a2x asciidoc man/compton.1 man/compton-trans.1 ; chmod +x a2x asciidoc
+PATH=$PWD:$PATH
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/compton/build]: Missing standalone `make`.

Comment (dylanaraps) [reply community/compton/build]: Standalone `make` should also allow you to remove the fake `asciidoc` portions.

Comment (a-schaefers) [reply community/compton/build]: unfortunately not the case for me

ompton.o opengl.o c2.o -lGL -lm -lrt -lpcre -lXcomposite -lXdamage -lXfixes -lXrender -lX11 -lXext -lXrandr -lXinerama -lconfig 
a2x --format manpage man/compton.1.asciidoc
make: a2x: Command not found
make: *** [Makefile:121: man/compton.1] Error 127
!> compton Build failed

Comment (dylanaraps) [reply community/compton/build]: Maybe a simple `sed` could suffice to change `a2x` to `:` (noop).

Comment (a-schaefers) [reply community/compton/build]: fixed

**Comment by a-schaefers** [comment]
this is now ready. was much simpler than we thought.

**Pull request merged**